kolibri/plugins/management/assets/src/actions.js

```diff
@@ -234,8 +234,8 @@ function showClassEditPage(store, classId) {
     ([users, cl]) => {
       const pageState = {
         modalShown: null,
-        classes: [cl],
-        users: users.map(_userState),
+        classrooms: [cl],
```

Comment:
let's try to find a more descriptive variable name than `cl`. cc @eli what did we use in the other place? even `class_` might work?